packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ExifHandler.java

```diff
@@ -0,0 +1,101 @@
+package io.flutter.plugins.imagepicker;
```

Comment:
Can we keep the file name the same for this PR? Renaming the file keeps the diff hard to review in Github.